defaults/main/sshd.yml

```diff
@@ -31,7 +34,8 @@ sshd_host_key_algorithms: >-
   ssh-ed25519-cert-v01@openssh.com,
   ssh-rsa-cert-v01@openssh.com,
   ssh-ed25519,
-  ssh-rsa,
```

Comment:
Apparently this key algorithm is based on SHA1, which is known to be broken.

Reply:
This makes sense (https://www.openssh.com/txt/release-8.7).

Reply:
but "The RFC8332 RSA SHA-2 signature algorithms rsa-sha2-256/512. These algorithms have the advantage of using the same key type as "ssh-rsa" but use the safe SHA-2 hash algorithms. These have been supported since OpenSSH 7.2 and are already used by default if the client and server support them."